README.md

```diff
@@ -170,6 +170,12 @@ If you are looking for the codebase of the current production version of GATK, p
    JAVA_OPTS="-Dsamjdk.use_async_io=false" ./gatk-launch <rest of command>
 ```
 
+* By default, GATK (non-spark) uses compression level 1 for writting BAM files (fastest code but least compressed files). To change the default, run GATK like this:
+
+```
+   JAVA_OPTS="-Dsamjdk.compression_level=5" ./gatk-launch <rest of command>
```

Comment:
By using `jinfo -sysprops` i verified that using `JAVA_OPTS` supplements `applicationDefaultJvmArgs`, eg `JAVA_OPTS="-Dsamjdk.compression_level=5"` only changes the compression level and not other properties.